scripts/ci/install-minikube.sh

```diff
@@ -57,22 +62,24 @@ uninstall() {
 
 stop() {
 	check_minikube
-	sudo $(which minikube) delete || true
-	sudo rm -rf ~/.minikube ~/.kube
+	sudo -E minikube delete || true
+	sudo rm -rf $HOME/.minikube $HOME/.kube
+	sudo rm -rf /root/.minikube /root/.kube
```

Comment:
why do you need/access to /root/.minikube /root/.kube as you using sudo -E

Reply:
We need both `kubectl get all` (as called vi CI scripts) as well as  `sudo kubectl get all` as called by integration tests to work. 

We create working kubectl (pointing to minikube) within /home/vagrant and then copy it over to `root`.